roles/pulp_health_check/README.md

```diff
@@ -0,0 +1,21 @@
+pulp_health_check
+=================
+
+Verify if Pulp Services are up and listening.
```

Comment:
Sorry for the late comment, but we *should* clarify that this role is meant to be run on the host that `pulp_api` is run against.